apps/dashboard/app/(app)/logs-v2/components/table/log-details/components/log-section.tsx

```
@@ -0,0 +1,71 @@
+      <div className="flex justify-between items-center">
+        <span className="text-[13px] text-accent-9 font-sans">{title}</span>
+      </div>
+      <Card className="bg-gray-2 border-gray-4 rounded-lg">
+        <CardContent className="py-2 px-3 text-xs relative group">
+          <pre className="flex flex-col gap-1">
+            {Array.isArray(details)
+              ? details.map((header) => {
+                  const [key, ...valueParts] = header.split(":");
+                  const value = valueParts.join(":").trim();
+                  return (
+                    <div className="group flex items-center w-full p-[3px]">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)